addons/survey/models/survey_survey_template.py

```diff
@@ -79,8 +82,9 @@ def action_load_sample_feedback_form(self):
         }).action_show_sample()
 
     @api.model
-    def action_load_sample_certification(self):
+    def action_load_sample_assessment(self):
         survey_values = {
+            'survey_type': 'assessment',
             'title': _('Certification'),
```

Comment:
Shouldn't the title follow the renaming? So here we would have "Assessment" as title instead of "Certification" ?